index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <link rel="stylesheet" href="style.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <div class="wrapper">
+    <div class="bar-nav">
+        <h1>My Logo</h1>
```

Comment:
- [ ] Kindly your logo should be inside the `a` tag and not an `h1` tag, so anyone who clicks on it switches him to the index page.

```
<a herf="index.html" class="logo">My Logo</a>
```

Reply:
Done